contract/assembly/model.ts

```diff
@@ -1,13 +1,15 @@
-import { storage } from "near-sdk-as";
+import { storage, u128 } from "near-sdk-as";
 
-const CHUNK_SIZE = 16;
-const CHUNK_COUNT = 5;
+export const CHUNK_SIZE = 16;
+export const CHUNK_COUNT = 5;
 const START_TILE_ID = "-1";
 
 @nearBindgen
 export class Chunk {
   nonce: i32;
   tiles: string[][];
+  owner: string;
```

Comment:
I think it's best to store owner and listing price outside of Chunk to avoid parsing / serialising `tiles` every time that ownership is changed.